packages/piro/test/MockModelEval_A_Tpetra.cpp

```diff
@@ -88,7 +88,7 @@ MockModelEval_A_Tpetra::MockModelEval_A_Tpetra(const Teuchos::RCP<const Teuchos:
 
     //set up jacobian graph
     crs_graph = rcp(new Tpetra_CrsGraph(x_map, vecLength));
-    std::vector<int> indices(vecLength);
+    std::vector<typename Tpetra_CrsGraph::global_ordinal_type> indices(vecLength);
     for (int i=0; i<vecLength; i++) indices[i]=i;
```

Comment:
Best practice is to make the cast explicit.  We've talked before about using a tagged integer type for GO, but that's future work.